forge-gui/res/cardsfolder/upcoming/antagonize.txt

```diff
@@ -0,0 +1,5 @@
+Name:Antagonize
+ManaCost:1 R
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target creature | NumAtt$ +4 | NumDef$ +3 | SpellDescription$ Target creature gets +4/+3 until end of turn.
```

Comment:
For very simple ValidTgts, you can skip TgtPrompt and Forge will parse it

No need to change this unless you feel like it, just fyi 😉